test/test_common/simulated_time_system_test.cc

```diff
@@ -22,12 +23,25 @@ class SimulatedTimeSystemTest : public testing::Test {
         start_monotonic_time_(time_system_.monotonicTime()),
         start_system_time_(time_system_.systemTime()) {}
 
-  void addTask(int64_t delay_ms, char marker) {
+  void trackPrepareCalls() {
+    base_scheduler_.registerOnPrepareCallback([this]() { output_.append(1, 'p'); });
+  }
+
+  void addTask(int64_t delay_ms, char marker, bool expect_monotonic = true) {
+    addCustomTask(
+        delay_ms, marker, []() {}, expect_monotonic);
```

Comment:
nit: fits on one line? I thought 'format' test would've failed due to this.

Reply:
I tried moving it to one line and then running the format script.  The format script undid my changes.